specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -0,0 +1,604 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Security Center",
+		"description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+		"version": "2019-01-01-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"security": [{
+		"azure_auth": ["user_impersonation"]
+	}],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/{scope}/providers/Microsoft.Security/assessments/{assessmentName}/subAssessments": {
```

Comment:
how is an assessment different from a sub-assessment? It may be better to have a quick 30 min sync to go over this and close. Is this already live on the service side?

Reply:
Had conversion with @sanjaiganesh :)

Reply:
@  you may add a summary for all.. example + reason

---
In reply to: [300073944](https://github.com/Azure/azure-rest-api-specs/pull/6422#discussion_r300073944) [](ancestors = 300073944)